projects/angular/src/utils/_normalize.scss

```diff
@@ -1,4 +1,4 @@
-/*! normalize.css v4.2.0 | MIT License | github.com/necolas/normalize.css */
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
There was an issue with the outdated normalize.css overwriting the output of the _input.clarity.scss mixins for placeholder color. Instead of just removing it I'm updating to the latest normalize.css.

Reply:
That change breaks testing so I removed the lines that were overwriting the _input.clarity.scss placeholder styles because they are not in the latest version of normalize (v8.x.x).